Libs/MRML/DisplayableManager/vtkMRMLCameraWidget.cxx

```diff
@@ -197,6 +198,29 @@ bool vtkMRMLCameraWidget::CanProcessInteractionEvent(vtkMRMLInteractionEventData
     return this->CanProcessButtonClickEvent(eventData, distance2);
     }
 
+  // Need to handle this event here because we keep handle generic event (rotation, translation,...)
+  if (widgetEvent == WidgetEventSetCrosshairPositionBackground && eventData->GetModifiers() == 0)
+    {
+    if (!this->GetCameraNode() || !this->GetCameraNode()->GetScene())
+      {
+      return true;
+      }
+
+    vtkMRMLScene* scene = this->GetCameraNode()->GetScene();
+    vtkMRMLCrosshairNode* crosshairNode = vtkMRMLCrosshairDisplayableManager::FindCrosshairNode(scene);
+    if (!crosshairNode || crosshairNode->GetCrosshairMode() != vtkMRMLCrosshairNode::Show3DCursor)
+      {
+      return true;
+      }
+
+    crosshairNode->FastPick3DOn();
+    this->ProcessSetCrosshair(eventData);
+    crosshairNode->FastPick3DOff();
+
+    distance2 = 1e10; // we can process this event but we let more specific widgets to claim it (if they are closer)
```

Comment:
```suggestion
    // After processing the event (see ProcessSetCrosshair), we artificially set the
    // distance to a large value, this may allow other displayable managers to process
    // the event using other widgets.
    distance2 = 1e10;
```